UPGRADE.md

```diff
@@ -80,6 +80,22 @@ helm upgrade --install --create-namespace --namespace kuma-system \
 kumactl install control-plane --set "legacy.transparentProxy=true" | kubectl apply -f-
 ```
 
+### Removal of deprecated options to reach applications bound to `localhost`
+
+The deprecated options `KUMA_DEFAULTS_ENABLE_LOCALHOST_INBOUND_CLUSTERS` and
+`defaults.enableLocalhostInboundClusters` were removed.
+
+Applications that are binding to `localhost` won't be reachable anymore.
```

Comment:
I'd add that this only affects transparent proxying

Reply:
done